docs/getting-started/products/ylockers/overview.md

```diff
@@ -1,8 +1,111 @@
+
+### Weights and Boosts
+
+The longer you stake, the greater your boost! Yearn's yLocker staking contract incentivizes long-term users by boosting their yield (up to a maximum of 2.5x).
+
+You'll reach max boost and achieve the maximum staking APR less than four weeks after depositing your yLocker tokens.
+
+To calculate your boost, the staking contract maintains a weight for every deposit (which is a function of the amount of yLocker tokens you have staked and the duration since it was staked).
+
+Your stake weight starts at 50 points with your initial deposit, and increases by 50 points each week (at 00:00:00 UTC, Thursday) until the maximum weight is reached after 4 weeks.
+
+**Let’s demonstrate with an example of how the weights work**
+
+In this example…
+
+* YearnBoostedStaker is deployed with maxGrowthWeeks = 4
+* You stake 100 yLocker tokens
+
+On deposit, your initial weight will be 50. At 00:00:00 UTC the following Thursday, your weight will increase to 100, then 150, then 200, and finally 250 (on the fourth Thursday following your deposit).
+
+| week             | balance | weight | boost |
+| ---------------- | ------- | ------ | ----- |
+| 0 (deposit week) | 100     | 50     | 0.5x  |
+| 1                | 100        | 100       | 1.0x      |
+| 2                | 100        | 150       | 1.5x      |
+| 3                | 100        | 200       | 2.0x      |
+| 4 (final growth week)               | 100        | 250       | 2.5x      |
+| 5 ...n                 | 100        | 250       | 2.5x      |
+
+To keep it simple, the example above does not address what happens if you make a withdrawal or a second deposit. If, for example, you deposit 100 Ylocker tokens every week for four weeks, you will then have four independent weight groups travelling through the system.
+
+A withdraw will always retrieve yLocker tokens from your most recent (least weighted) deposit, leaving the higher weighted yLocker tokens to continue along.
+
+Your total weight is equal to the sum of each of your deposit’s weight. And the total system weight is the sum of all user weight.
+
+### Rewards and Claiming
+
+Each week, Yearn claims its share of protocol fees and vote-maximized bribes. These are swapped for ecosystem stablecoins, wrapped as a yield-bearing Yearn V3 vault token, and deposited directly into the reward distributor contract.
+
+yLocker stakers are entitled to the following rewards:
+
+* 100% of their share of protocol fees
+* 90% of their share of gauge votes will be used to earn vote-maximized bribes
+  * *(the other 10% votes to support yLocker token liquidity)*
+
+Your rewards accrue week over week and are never lost if unclaimed. In fact, they begin earning you additional stablecoin yield from the moment we receive them! When claimed, Yearn V3 vault tokens will be received directly to your wallet.
+
+To reward early adopters, yLocker stakers may receive additional rewards during the launch phase.
```

Comment:
I would probably kill this sentence, `To reward early adopters, yLocker stakers may receive additional rewards during the launch phase.` 

it's a nebulous promise that will probably just get us in more trouble than any benefit it offers

Reply:
step away, he's got tough toes!

but seriously, I do know where that point came from...and I know what he's getting at, but we should probably be more specific if we're going to include something about it

Reply:
ok. going to mark as resolved and if there are additional rewards added I can add them in here.